spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -196,6 +196,19 @@ the sample below postgresql has been chosen.
 1. Save the changed pom.xml
 2. Build the application as described here: <<appendix-building.adoc#building, Building Spring Cloud Data Flow>>
 
+==== Task Application Repository
+
+When launching a task application be sure that the database driver that is being
+used by Spring Cloud Dataflow is also a dependency on the task application.  For
+example if your Spring Cloud Dataflow is set to use Postgresql be sure that
+the task application also has Postgresql as a dependency.
+
+NOTE: When executing tasks externally (i.e. command line) and you wish for
+Spring Cloud Dataflow to show the TaskExecutions in its UI, be sure that the
+task's datasource properties are set to the same as Dataflow's.   By default
```

Comment:
> be sure that the task's datasource properties are set to the same as Dataflow's

In order to avoid the standalone term "dataflow", could we rephrase it to: "be sure that a common datasource settings are shared among the both"?